bin/vl2vg

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+// Compile a vegalite spec to vega
+
+'use strict';
+
+var helpText =
+  'Compile a vega-lite spec to vega.\n\n' +
+  'Usage: vl2vg vega_lite_json_file [data_stats] [-p]\n\n' +
+  'data_stats only has to be set if the data is not embedded in the vega-lite json file. '+
+  'You will get an error if you forget to provide statistics and do not embed data into the spec.';
+
+// import required libraries
+var fs = require('fs'),
+    vl = require('../src/vl'),
+    json3 = require('../lib/json3-compactstringify.js');
```

Comment:
Ah, I should properly create a repo for this at some point.  This is still hacky.